config/application.rb

```diff
@@ -12,7 +12,6 @@
 # require "action_text/engine"
 require "action_view/railtie"
 require "action_cable/engine"
-require "sprockets/railtie"
```

Comment:
[Auto generated change]
We don't use the sprockets on default.
So I added the `sprockets-rails` here: https://github.com/ranguba/ranguba/pull/8.

Reply:
TODO: We will deal with this change in another PR.

Reply:
We shouldn't accept this change if we keep using `Sprockets`.
This `app:update` task will automatically remove it so we will explicitly require it in Gemfile at the following PRs. 
- https://github.com/ranguba/ranguba/pull/14